### PR TITLE
Update language file for zh-Hant

### PR DESCRIPTION
**Pull request by mics8128**
Update language file for zh-Hant

### CODE REVIEW

File: src/resources/lang/zh-Hant/crud.php [hunk 1]
```diff
@@ -39,10 +39,15 @@
     'to'                   => '到',
     'undo'                 => '回復',
     'revision_restored'    => '成功回復修訂版本',
+    'guest_user'           => '遊客',
+
+    // Translatable models
+    'edit_translations' => 'EDIT TRANSLATIONS',
```
Comment (niladam) [inline src/resources/lang/zh-Hant/crud.php]: Why is this still untranslated ?

Comment (lloy0076) [reply src/resources/lang/zh-Hant/crud.php]: It's rather ironic that `edit translations` is the one that is **not** translated if I may say so myself 🙄

**Comment by niladam** [comment]
@lloy0076 - yep. My point exactly:)

File: src/resources/lang/zh-Hant/crud.php (hunk 1 above)
Comment (niladam) [reply src/resources/lang/zh-Hant/crud.php]: @lloy0076 - yep, my point exactly:)

Comment (mics8128) [reply src/resources/lang/zh-Hant/crud.php]: i miss that part :P
updated.

Comment (tabacitu) [reply src/resources/lang/zh-Hant/crud.php]: Haha. "edit_translations", out of all strings :-) Nice. Thanks guys!

**Comment by tabacitu** [comment]
Thank you @mics8128! Merged!

**Pull request merged**